File: greg/day3/twosum.rs
impl Solution {
    pub fn two_sum(nums: Vec<i32>, target: i32) -> Vec<i32> {
        use std::collections::HashMap;
        let mut map = HashMap::new();
        for (i, n) in nums.iter().enumerate() {
            let diff = target - n;
            if map.contains_key(&diff) {
                return vec![*map.get(&diff).unwrap() as i32, i as i32];
            }
            map.insert(n, i);
        }
        return vec![];
    }
}

File: Jaxon/Day3 - Two Sum/TwoSum.py
class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:

        hashmap = {}
        for i, num in enumerate(nums):

            if (target - num) in hashmap:
                return [i,hashmap[target-num]]
                
            hashmap[num] =  i

            